llvm/include/llvm/Transforms/IPO/Attributor.h

```diff
@@ -6120,6 +6170,9 @@ struct AAPointerInfo : public AbstractAttribute {
   virtual const_bin_iterator begin() const = 0;
   virtual const_bin_iterator end() const = 0;
   virtual int64_t numOffsetBins() const = 0;
+  virtual void dumpState(raw_ostream &O) const = 0;
```

Comment:
Like I mentioned before, there are pure virtual functions, so we will want to mark the c'tor of `AAPointerInfo` to be `protected`.